integration_tests/e2e/goal/review.cy.ts

```diff
@@ -0,0 +1,119 @@
+import Page from '../../pages/page'
+import CreateGoalPage from '../../pages/goal/CreateGoalPage'
+import AddStepPage from '../../pages/goal/AddStepPage'
+import AddNotePage from '../../pages/goal/AddNotePage'
+import ReviewPage from '../../pages/goal/ReviewPage'
+import OverviewPage from '../../pages/overview/OverviewPage'
+
+context('Review goal(s)', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('getPrisonerById', 'G6115VJ')
+    cy.task('getPrisonerById', 'H4115SD')
+  })
+
+  it('should not be able to navigate directly to Review Goal given previous forms have not been submitted', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+
+    // When
+    cy.visit(`/plan/${prisonNumber}/goals/add-note`)
```

Comment:
This is navigating to add note, not review goal ?
(I think the rest of the test is sound; its just the `cy.visit` that is wrong?